Source/FlowEditor/Private/Graph/Nodes/FlowGraphNode.cpp

```diff
@@ -276,6 +282,13 @@ void UFlowGraphNode::InsertNewNode(UEdGraphPin* FromPin, UEdGraphPin* NewLinkPin
 
 void UFlowGraphNode::ReconstructNode()
 {
+	if (bIsReconstructingNode)
```

Comment:
Prevent infinite recursion if the reconstruction triggers another reconstruction (somehow).

I fixed the case that was doing an infinite reconstruction recursion, but I left in this protection.